release.sh

```diff
@@ -11,5 +11,6 @@ cd ../guides.emberjs.com
 mkdir $VERSION
 mv ../guides/build snapshots/$VERSION
 node tasks/update-versions
-
-# NEEDS TO UPDATE _redirects
\ No newline at end of file
+sed -Ei .bak "s/v[0-9]+[.][0-9]+[.][0-9]+/$VERSION/g" snapshots/_redirects
```

Comment:
Nice work here

Reply:
Are .bak files part of gitignore?

Reply:
I think we can remove `.bak` to avoid the temporary file.